test/contestant_test.rb

```diff
@@ -0,0 +1,76 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require "./lib/contestant"
+require "./lib/game"
+require "pry"
+
+class ContestantTest < Minitest::Test
+
+  def test_it_exist
+    alexander = Contestant.new({first_name: 'Alexander',
+      last_name: 'Aigiades',
+      age: 28,
+      state_of_residence: 'CO',
+      spending_money: 10})
+
+    assert_instance_of Contestant, alexander
+  end
+
+  def test_has_attributes
+    alexander = Contestant.new({first_name: 'Alexander',
+      last_name: 'Aigiades',
+      age: 28,
+      state_of_residence: 'CO',
+      spending_money: 10})
+
+    assert_equal "Alexander Aigiades", alexander.full_name
+    assert_equal 28, alexander.age
+    assert_equal "CO", alexander.state_of_residence
+    assert_equal 10, alexander.spending_money
+  end
+
+  def test_can_check_out_of_state
+    alexander = Contestant.new({first_name: 'Alexander',
+      last_name: 'Aigiades',
+      age: 28,
+      state_of_residence: 'CO',
+      spending_money: 10})
+
+    assert_equal false, alexander.out_of_state?
```

Comment:
Even though the interaction pattern doesn't call for it, it's always a good idea to test both a true and false return for all boolean methods.